http/httpclient/client.go

```diff
@@ -774,23 +774,25 @@ func addUserAgentHeader(req *http.Request) {
 }
 
 type DownloadFileDetails struct {
-	FileName      string `json:"FileName,omitempty"`
-	DownloadPath  string `json:"DownloadPath,omitempty"`
-	RelativePath  string `json:"RelativePath,omitempty"`
-	LocalPath     string `json:"LocalPath,omitempty"`
-	LocalFileName string `json:"LocalFileName,omitempty"`
-	ExpectedSha1  string `json:"ExpectedSha1,omitempty"`
-	Size          int64  `json:"Size,omitempty"`
+	FileName        string `json:"FileName,omitempty"`
+	DownloadPath    string `json:"DownloadPath,omitempty"`
+	RelativePath    string `json:"RelativePath,omitempty"`
+	LocalPath       string `json:"LocalPath,omitempty"`
+	LocalFileName   string `json:"LocalFileName,omitempty"`
+	ExpectedSha1    string `json:"ExpectedSha1,omitempty"`
+	Size            int64  `json:"Size,omitempty"`
+	IgnoreHashCheck bool   `json:"IgnoreHashCheck"`
```

Comment:
I avoided this since I've had problems with omitempty when dealing with default true options (i.e. false and empty are the same thing). However, in this case that clearly isn't a problem since the value defaults to false. I'll correct.